Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp

```diff
@@ -18,6 +17,7 @@
 #include <QDesktopServices>
 #include <QUrl>
 #include <fstream>
+#include "MainWindow.h"
```

Comment:
We do not want plugins or item to include "MainWindow.h". `MainWindow::lock_test_item` should be moved to an interface.